traefik-crds/crds-files/traefik/traefik.io_ingressroutes.yaml

```diff
@@ -418,4 +459,4 @@ spec:
         - spec
         type: object
     served: true
-    storage: true
```

Comment:
Minor nitpick: I have no idea what caused this, but it seems the empty line at the end of the file here and in a lot of other CRDs were removed, causing "incorrect" diffs. Probably we should aim to keep those around and also ensure any tooling that updates these files should always behave that way.